src/components/TableCell/TableCell.tsx

```diff
@@ -1,17 +1,9 @@
 import clsx from 'clsx';
 import type { ReactNode } from 'react';
 import React from 'react';
-import styles from '../Table/Table.module.css';
+import styles from './TableCell.module.css';
 
-export interface Props {
-  /**
-   * The alignment of the text in the cell
-   */
-  align?: 'insetLeft' | 'left' | 'right' | 'center';
```

Comment:
Natively deprecated attribute, shouldn't be encouraging, just style with classes